### ISSUE TITLE
[Dropdown][A11Y] No aria-expanded found when component is in the default state 

### ISSUE THREAD

**Issue by djalmaaraujo** [radix-ui/primitives#1633]
### Current Behavior

When the Dropdown is closed, the `aria-expanded` is not present as `false`, which means that the screen reader will not announce the correct state of the Dropdown. 

If you activate, it adds the correct `aria-expanded` as `true`.

### Expected behavior

It should always display the current state of the `aria-expanded`.

### Reproducible example

The Documentation website is already a good example:
https://www.radix-ui.com/docs/primitives/components/dropdown-menu

### Suggested solution

If it's a Dropdown/Modal, etc, adds `aria-expanded` based on the `defaultOpen` property or any related property of that element.

### Additional context

N/A

### Your environment

Regular Chrome Browser — Version 104.0.5112.101 (Official Build) (arm64)
Radix latest version: 1.0.0.

**Comment by benoitgrelard** [comment]
Yeah I think that's correct, we seem to do this for all other components so not sure why this one was skipped.

**Comment by djalmaaraujo** [comment]
@benoitgrelard Thank you! Where I can see the pre-releases? Is it possible to use this already?

**Comment by benoitgrelard** [comment]
You can install them with the `@next` tag, in this case here is the latest one: https://www.npmjs.com/package/@radix-ui/react-dropdown-menu/v/1.0.1-rc.3
